src/research/pq-verkle.md

```diff
@@ -0,0 +1,84 @@
+Below is a structured summary of the driving motivation behind this research. Verkle has significant benefits over the alternative storage proposals. Nevertheless, it has some drawbacks too like Pedersen Hash speeds and no PQ-security.
+
+Thus, one area of research that is active and stateless consensus is collaborating on together with EF Cryptography team is PQ-Verkle trees.
```

Comment:
```suggestion
Thus, one area of research that is active and Stateless Consensus team is collaborating on together with EF Cryptography team is PQ-Verkle trees.
```

Reply:
Thus, PQ-Verkle trees are an active area of research, and Stateless Consensus is working on it, along with the EF Cryptography team.